docs/plugins/repository-s3.asciidoc

```diff
@@ -190,8 +190,12 @@ The following settings are supported:
 
 `storage_class`::
 
-    Sets the S3 storage class type for the backup files. Values may be
+    Sets the S3 storage class for snapshot repository objects. Values may be
     `standard`, `reduced_redundancy`, `standard_ia`. Defaults to `standard`.
+    All new objects created will use the set storage class.
```

Comment:
I think this sentence can be removed - the subsequent sentence says this and more.